modules/cli-installing-cli-rpm.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/getting-started.adoc
+
+[id="cli-installing-cli-rpm_{context}"]
+= Installing the CLI by using an RPM
+
+For Red Hat Enterprise Linux (RHEL), you can install the OpenShift CLI (`oc`) as an RPM if you have an active OpenShift Container Platform subscription on your Red Hat account.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+----
+# subscription-manager register
```

Comment:
If you are showing them using `oc` as root user, do you need to indicate this as a prereq?

Reply:
Good suggestion